src/content/translations/fil/guides/how-to-create-an-ethereum-account/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Paano "gumawa" ng Ethereum account
+description: Isang step-by-step na gabay sa paggawa ng Ethereum account gamit ang isang wallet.
+lang: fil
+---
+
+# Paano "gumawa" ng Ethereum account
+
+Maaaring gumawa ng Ethereum account nang libre ang kahit sino, kahit kailan. May ilang paraan, ngunit ang pinakamadali at pangkaraniwang paraan ay gamitin ang isang app na kilala bilang wallet. Gumagawa at sine-secure ng mga wallet ang mga key na nagbibigay-daan sa iyong gamitin ang Ethereum. Puwede mong gamitin ang wallet mo para magpadala ng mga transaksyon, tingnan ang mga balanse ng iyong token, at kumonekta sa mga apps na ginawa sa Ethereum, gaya ng mga token exchange, laro, mga NFT marketplace, at iba pa. Pinapayagan ka na rin ngayon ng ilang "web2" app na mag-sign in sa Ethereum.
+
+Hindi tulad ng pagbubukas ng bagong account sa isang kumpanya, ang paggawa ng Ethereum account ay malaya, pribado, at hindi kailangan ng pahintulot. Ang mga account ay kontrolado ng mga key na ginagawa mo sa tulong ng wallet mo, at hindi ibinibigay ng third party, o naka-store sa isang central registry.
+
+## Step 1: Pumili ng wallet
+
+Ang wallet ay isang app na tumutulong sa iyong pamahalaan ang iyong Ethereum account. Ginagamit nito ang iyong mga key para magpadala at tumanggap ng mga transaksyon at mag-sign in sa mga app. Maraming iba't ibang wallet na mapagpipilian—mobile, desktop, o kahit mga browser extension.
+
+<ButtonLink to="/wallets/find-wallet/">
+  Maghanap ng wallet
+</ButtonLink>
+
+Kung bago ka, maaari mong piliin ang filter na “Baguhan sa crypto” sa page na "maghanap ng wallet" upang matukoy ang mga wallet na naglalaman ng lahat ng kinakailangang feature na angkop para sa mga baguhan.
+
+![pagpili ng filter sa page na 'maghanap ng wallet'](./wallet-box.png)
```

Comment:
<img width="911" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/3fc4d55d-873b-4cf4-ad5a-7fd2217268fb">

Images not loading

Reply:
Uncertain why... syntax is identical to other locales that are currently working just fine. Merge updated with `dev` and will let it rebuild first before moving further

Reply:
@corwintines @pettinarip Rebuilt and this is still showing up this way... any idea what could be going on here?

Reply:
I think this has to do with `gatsby-remark-fix-static-urls` only doing 2 letter lang codes. Let me look at this, think I know whats going on.